lib/cc/cli/init.rb

```diff
@@ -10,7 +10,7 @@ class Init < Command
       def run
         if !upgrade? && filesystem.exist?(CODECLIMATE_YAML)
           warn "Config file .codeclimate.yml already present.\nTry running 'validate-config' to check configuration."
-          create_default_configs
+          create_default_engine_configs
```

Comment:
Bummer: I was doing a fair amount of rebase editing & must have fouled something. Sorry, I'll redo.

> I still can't see where you fallback to config-generator when existing_cc_config doesn't exist, but maybe I'm misunderstanding the intent (because of the diff confusion?).

I don't think that case exists, and I did not intend to implement that behavior. If the YAML already exists, it already exists, yay. If it does not exist when you run `init`, `#create_codeclimate_yaml` will write it, and again it will exist before `#create_default_engine_configs` is called.